src/common/utils.tsx

```diff
@@ -412,3 +417,8 @@ export function getContractId(
     ? transaction.contract_call.contract_id
     : undefined;
 }
+
+export const isClarinetInstance = (url: string) => url.includes('.devnet:3999');
```

Comment:
Is there scope for abuse in this check? 🤔  Would it be better/safer to pass url to `new URL()` and check host?